README.md

```diff
@@ -1,16 +1,41 @@
-# Shelf.nu - Open Source Asset Management Infrastructure for everyone.
 
-[![Twitter](https://img.shields.io/twitter/url/https/twitter.com/cloudposse.svg?style=social&label=Follow%20%40ShelfQR)](https://twitter.com/ShelfQR)
-[![🚀 Deploy](https://github.com/Shelf-nu/shelf.nu/actions/workflows/deploy.yml/badge.svg?branch=dev)](https://github.com/Shelf-nu/shelf.nu/actions/workflows/deploy.yml)
-
-Shelf 🏷️ Asset Management infrastructure for absolutely everyone (open source).
 
+# Shelf.nu  
+<a href="https://www.shelf.nu/" target="_blank">
+<img width="100%" src="https://cdn.discordapp.com/attachments/1143882281069592658/1143949082017153044/Frame_102.jpg" />
+</a>
+<h4 align="center">
+✨ Open Source Asset Management Infrastructure for everyone. ✨
+</h4>
+<p align="center" >
+Shelf 🏷️ Asset Management infrastructure for absolutely everyone (open source).<br/> <br/>
 Shelf is a simple and visual asset management and location tracking system that allows people to track their physical assets with ease.
+</p>
 
-## Core Features and Benefits
+## Core Features and Benefits 💫
 
 With Shelf, you can take a picture of any item you own and store it in your own database. From there, you can generate a printable code (QR) that you can tag onto the item, making it easy to identify and locate in the future. Shelf has a handy code printing area where you can add as many QR codes as you can on an A4 sticker paper sheet. You can also add detailed information about the item, including its purchase date, purchase price, warranty information, and more.
 
+
+
+<div align="left">
+
+
+
+<p align="center">
+    <a href="https://www.shelf.nu/"><b>Website</b></a> •
+    <a href="https://github.com/Shelf-nu/shelf.nu"><b>Documentation</b></a> •
```

Comment:
Lets add `/docs`. We don't have it now but as soon as we merge this it will be there so we are good.